lib/show.rb

```diff
@@ -0,0 +1,15 @@
+class Show
+
+attr_reader :name, :creator
+
+  def initialize(name, creator, character)
+    @name = name
+    @creator = creator
+  end
+
+  def characters
+  
+
+  end
+
+end
```

Comment:
good start in here; since your initializer was being given a list of characters you could save that as an instance variable in your initialize method and then make an attr_reader from that